e2e/example.spec.ts

```
@@ -0,0 +1,4 @@
+import { test } from '@playwright/test';
+
+// eslint-disable-next-line no-empty-pattern, @typescript-eslint/no-empty-function
+test('empty test', async ({}) => {});
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)